decidim-assemblies/app/views/decidim/assemblies/admin/assemblies/index.html.erb

```diff
@@ -29,12 +29,14 @@
   </div>
 
   <div class="mt-4">
```

Comment:
I think we could add the DIV into the permission check. Otherwize, we would have an empty div "left" around

![image](https://github.com/user-attachments/assets/3d7a3e38-4605-436f-a230-68a260b9b687)